exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,31 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var total = numberOfStudents + numberOfMentors;
+// console.log(total);
+var percentageOfStudents = Math.round((numberOfStudents / total) * 100);
+console.log(percentageOfStudents);
+
+var percentageOfMentors = Math.round((numberOfMentors / total) * 100);
+console.log(percentageOfMentors);
+console.log("Percentage of students " + percentageOfStudents + " Percentage of mentors " + percentageOfMentors);
+console.log(`Percentage of students ${percentageOfStudents} Percentage of mentors ${percentageOfMentors}`);
+
+function percentageStudents(numberOfStudents, total){
+    return (numberOfStudents / total) * 100;
+
+}
+
+
+// console.log(percentageStudents(15,23));
```

Comment:
Hi Bisi! 

It's normally best to remove the commented out code that you don't need - it makes it much easier to read

https://syllabus.codeyourfuture.io/guides/code-style-guide/#dont-leave-lots-of-commented-out-code